modules/340-monitoring-kubernetes/images/kubelet-eviction-thresholds-exporter/Dockerfile

```diff
@@ -11,4 +11,5 @@ RUN apk add --no-cache git && \
 FROM $BASE_DISTROLESS
 
 COPY --from=artifact /src/loop /
+
```

Comment:
```suggestion
USER 64535
```

Reply:
It looks like the user directive is missing.

Reply:
As previously agreed, this directive has been moved to the base image and removed from the final images